src/en/articles/landmarks/index.njk

```diff
@@ -217,20 +216,20 @@ Using ARIA landmarks to identify regions of a page</a>.</p>
   </li>
 
   <li>
-    <p>All page content must be included in <abbr>HTML5</abbr> tags of in <abbr>ARIA</abbr> structure landmarks. This allows users to access all content, even that which they may not have found outside these tags.</p>
+    <p>All page content must be included in <abbr>HTML5</abbr> tags of in <abbr>ARIA</abbr> structure landmarks. This allows users to access all content, even the one not easily discoverable, because outdside these tags.</p>
```

Comment:
Yes, clearer !

What about: 

> I'm just correcting here "This allows users to access all content**s**, even the ones not easily discoverable, because **outside of** these tags".

OR  : 

> "This allows users to access **the entire** content, even the **parts** not easily discoverable, because **outside of** these tags".

ℹ️ I put here the modifications I suggest in bold. They won't be in bold in the article.